apps/microtvm/arduino/template_project/microtvm_api_server.py

```diff
@@ -337,7 +332,7 @@ def _copy_cmsis(self, project_path: pathlib.Path, cmsis_path: str):
         However, the latest release does not include header files that are copied in this function.
         """
         (project_path / "include" / "cmsis").mkdir()
-        cmsis_path = get_cmsis_path(cmsis_path)
+        cmsis_path = pathlib.Path(cmsis_path)
```

Comment:
this one also removes the possibility of setting CMSIS_PATH through env right? can you add to PR description?

Reply:
not it doesn't, the default value for cmsis_path is set by environment. Check `python/tvm/micro/project_api/server.py` where we defined the cmsis_path as a projectOption

Reply:
ah, gotcha, nevermind then